### ISSUE TITLE
horizontal scroll

### ISSUE THREAD

**Issue by aleixsuau** [dinony/od-virtualscroll#5]
Hi Dinony,
Do you plan horizontal scrolling?
Thanks for your cool library!

**Comment by dinony** [comment]
Hey, thanks! Horizontal scrolling is definitely on the radar. I hope I will have some spare time to add this feature in the next weeks.

**Comment by manju-reddys** [comment]
Hi, Any plan to add it horizontal scroll soon?

**Comment by dinony** [comment]
Yes, I've allocated some time to get basic [maintenance](https://github.com/dinony/od-virtualscroll/milestone/2) done in the next weeks. After that I'm pretty sure that this will be the feature I will focus on.

**Comment by manju-reddys** [comment]
@aleixsuau  and @dinony , I have implemented a directive [here](https://plnkr.co/edit/K0SSAxlLOfukNVAIcd3t?p=preview) that handle unlimted records both horizontal and vertical (default). But variable height is not possible to support in this logic.
